sky/tools/create_macos_framework.py

```diff
@@ -220,7 +220,7 @@ def zip_framework(dst, args):
 def zip_xcframework_archive(dst):
   filepath_with_entitlements = ''
   filepath_without_entitlements = (
-      'FlutterMacOS.xcframework/macos-arm64_x84_64/'
+      'FlutterMacOS.xcframework/macos-arm64_x84_64/FlutterMacOS.framework/Versions/A/FlutterMacOS'
```

Comment:
Ooof, this was my bad. It's not that the path isn't complete, it's that it needs to be `x86`, not `x84` 

```suggestion
      'FlutterMacOS.xcframework/macos-arm64_x86_64/'
```

Reply:
Thanks for the catch! Do you happen to have a built zip file of xcframework that I could take a look? (just to double check its file structure)

I thought if there is a binary inside the xcframework folder we would use the full path to the binary

Reply:
Here's an example: https://storage.googleapis.com/flutter_infra_release/flutter/10331db8f748432d901d566b72ad0ab90f6cd1b9/darwin-x64/framework.zip

Reply:
Perfect, yeah this would work!